### PR TITLE
Update microsoft-team.md

### PR DESCRIPTION
**Pull request by richlander**

### CODE REVIEW

File: Documentation/microsoft-team.md [hunk 1]
```diff
@@ -4,22 +4,39 @@ If you work on or with the .NET Team, you will need to onboard into various GitH
...
 
-You need to join teams in two organizations. Click the two links:
+## Security best practices
+
+Bad actors try to break into our accounts all the time (see ["failed login attempts" on your account](https://github.com/settings/security-log?q=action%3Auser.failed_login)). You need to apply the following guidance to (A) stay secure, and (B) maintain access to your account.
+
+The following best practices are required for org owners, and recommended for repo admins.
+
+* Do not use [SMS](https://en.wikipedia.org/wiki/SIM_swap_scam) for 2FA or as a recovery fallback (disable those options).
+* Do use the [GitHub mobile app](https://github.blog/2022-01-25-secure-your-github-account-github-mobile-2fa/) for 2FA.
```
Comment (mmitche) [inline Documentation/microsoft-team.md:19]: I would reword this as something like:

Register at least two two-factor authentication methods of the folllowing
- [GitHub mobile app](https://github.blog/2022-01-25-secure-your-github-account-github-mobile-2fa/)
- Register a [security key(s)](https://www.yubico.com/works-with-yubikey/catalog/github/)
- An authenticator app 

I have an authenticator app and Yubikey, but I don't use GitHub mobile.

Comment (richlander) [reply Documentation/microsoft-team.md:19]: Good call.

Comment (richlander) [reply Documentation/microsoft-team.md]: https://github.com/dotnet/core/pull/7418/commits/2b9ef058be30833f5a01ea4f1088f27607bb8426
